frontend/src/app/(pages)/questions/[id]/page.tsx

```diff
@@ -1,18 +1,32 @@
-"use client";
+import { getQuestionById } from "@/helpers/question/question_api_wrappers";
 import Question from "@/types/question";
-import { getQuestionById } from "../../../../helpers/questions/services";
-import { useParams } from "next/navigation";
+import { notFound } from "next/navigation";
+import { Suspense } from "react";
 
-export default async function QuestionDetailPage() {
-  const params = useParams();
-  const question: Question = await getQuestionById(params.id as string);
+async function getQuestion(id: string) {
+  const res = await getQuestionById(id, 'no-cache');
```

Comment:
Is there a reason why this should be no-cache? I am not very familiar with caching, I know in nextjs every `fetch()` will be cached automatically, so if we set it to no-cache we will always get data from the server, but why we want the question to be loaded every time from the server? Does it mean that if we cached the data, then even someone update the question, we will not be able to see the latest changes?

Reply:
this is equivalent to `getserversideprops` in the app router.
see this https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#server-side-rendering-getserversideprops.

Reply:
I see. Yes, we want `getServersideProps` behaviour. Thanks for explaining!